deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -7,16 +7,135 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
     repository: https://github.com/openshift/windows-machine-config-operator
-  name: windows-machine-config-operator.v0.0.0
+  name: windows-machine-config-operator.v1.0.0-rc1
   namespace: openshift-windows-machine-config-operator
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions: {}
-  description: Placeholder description
+  description: |-
+    Issues with this distribution of WMCO can be opened against the [WMCO repository](https://github.com/openshift/windows-machine-config-operator).
```

Comment:
Given this is for the release it would need to point to [Bugzilla](https://bugzilla.redhat.com/buglist.cgi?product=OpenShift%20Container%20Platform&component=Windows%20Containers&resolution=---).